django_mongodb/features.py

```diff
@@ -59,6 +78,19 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         "model_fields.test_jsonfield.TestQuerying.test_order_grouping_custom_decoder",
         "model_fields.test_jsonfield.TestQuerying.test_ordering_by_transform",
         "model_fields.test_jsonfield.TestQuerying.test_ordering_grouping_by_key_transform",
+        # DecimalField lookup with F expression crashes:
+        # decimal.InvalidOperation: [<class 'decimal.ConversionSyntax'>]
+        "lookup.tests.LookupTests.test_lookup_rhs",
+        # Wrong results in queries with multiple tables.
+        "annotations.tests.NonAggregateAnnotationTestCase.test_annotation_aggregate_with_m2o",
+        "annotations.tests.NonAggregateAnnotationTestCase.test_annotation_reverse_m2m",
+        "annotations.tests.NonAggregateAnnotationTestCase.test_chaining_annotation_filter_with_m2m",
+        "lookup.tests.LookupTests.test_lookup_collision",
+        "expressions_case.tests.CaseExpressionTests.test_join_promotion",
+        "expressions_case.tests.CaseExpressionTests.test_join_promotion_multiple_annotations",
+        "ordering.tests.OrderingTests.test_order_by_grandparent_fk_with_expression_in_default_ordering",
+        "ordering.tests.OrderingTests.test_order_by_parent_fk_with_expression_in_default_ordering",
+        "ordering.tests.OrderingTests.test_order_by_ptr_field_with_default_ordering_by_expression",
```

Comment:
In what way are these results wrong?

Reply:
I think there is more than one issue represented here. I'll investigate in more detail later but don't want to hold up the merge for that.

Reply:
Agreed.